### PR TITLE
Add "notes" section from https://mrchromebox.tech/#devices

### PR DESCRIPTION
**Pull request by ethanaobrien**
This pr adds the notes section, from https://mrchromebox.tech/#devices, as well as the CELES post install script (instead of linking to a deleted post)

I'm really bad at writing docs most of the time, so if anything needs to be changed, let me know!

### CODE REVIEW

File: docs/supported-devices.md [hunk 1]
```diff
@@ -3331,6 +3331,24 @@ If your device isn't listed, it doesn't have custom firmware support.
...
+
+<ol>
+<li id="cite_note-stumpy-1"><b>STUMPY</b>: The DisplayPort outputs don't seem functional under Windows, but work fine pre-OS and under Linux. <br>VGA output (via DVI port) is not functional.</li>
+<li id="cite_note-celes-2"><b>CELES</b>: Running Windows on CELES requires a post-install workaround. (see below) <br>
+CELES can be a highly problematic device running OSes other than ChromeOS. <b>Highly recommended to avoid</b>.</li>
+<li id="cite_note-stoney-3"><b>AMD</b>: Some StoneyRidge devices (CAREENA? BARLA?) may not currently have functional Legacy Boot Mode due to bugs in Google's firmware.</li>
+<li id="cite_note-glk-4"><b>GeminiLake</b> legacy boot mode/alternative firmware menu works but does not show payload list</li>
+</ol>
+
+### CELES post install script
```
Comment (meghan06) [inline docs/supported-devices.md:3345]: This should be moved to post-install.md

Comment (meghan06) [inline docs/supported-devices.md:3345]: This should be moved to post-install.md, under Windows.

Comment (ethanaobrien) [reply docs/supported-devices.md]: Moved it!

File: docs/supported-devices.md [hunk 2]
```diff
@@ -3336,12 +3336,19 @@ If your device isn't listed, it doesn't have custom firmware support.
...
-<li id="cite_note-celes-2"><b>CELES</b>: Running Windows on CELES requires <a rel="nofollow noopener noreferrer" class="external text" href="https://www.reddit.com/r/chrultrabook/comments/9vi9kv/windows_10_on_samsung_chromebook_3_celes_fix/" target="_blank">a post-install workaround</a>. <br>
+<li id="cite_note-celes-2"><b>CELES</b>: Running Windows on CELES requires a post-install workaround. (see below) <br>
 CELES can be a highly problematic device running OSes other than ChromeOS. <b>Highly recommended to avoid</b>.</li>
 <li id="cite_note-stoney-3"><b>AMD</b>: Some StoneyRidge devices (CAREENA? BARLA?) may not currently have functional Legacy Boot Mode due to bugs in Google's firmware.</li>
 <li id="cite_note-glk-4"><b>GeminiLake</b> legacy boot mode/alternative firmware menu works but does not show payload list</li>
 </ol>
 
+### CELES post install script
+
+1. Boot the windows install media you used to install windows.
```
Comment (joebobbio) [inline docs/supported-devices.md]: Please also remember that Windows is a proper noun (nitpick).

Comment (ethanaobrien) [reply docs/supported-devices.md]: You're right, lol. I'll fix that

Comment (meghan06) [reply docs/supported-devices.md]: I was going to point that out

Comment (ethanaobrien) [reply docs/supported-devices.md]: Oh my, I'm sorry I completely missed the second Windows. Sorry

**Pull request merged**